src/realm/util/file_mapper.hpp

```diff
@@ -72,8 +72,11 @@ inline void set_page_reclaim_governor_to_default()
 // Retrieves the number of in memory decrypted pages, across all open files.
 size_t get_num_decrypted_pages();
 
-// Retrieves the amount of memory used for decrypted pages, across all open files.
-size_t get_decrypted_memory_size();
+// Retrieves the 
+// - amount of memory used for decrypted pages, across all open files.
+// - current target for the reclaimer (desired number of decrypted pages)
+// - current workload size for the reclaimer, across all open files.
+void get_decrypted_memory_stats(size_t& memory_size, size_t& reclaimer_target, size_t& reclaimer_workload);
```

Comment:
How about returning a struct with the 3 numbers?

Reply:
good idea.